src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TaskListViewModel.cs

```diff
@@ -48,11 +52,183 @@ public string SectionStatus(int sequenceId, int sectionId)
             return RoatpTaskListWorkflowService.SectionStatus(ApplicationSequences, NotRequiredOverrides, sequenceId, sectionId, ApplicationRouteId);
         }
 
+        public string WhosInControlCss
+        {
+            get
+            {
+                var status = WhosInControlSectionStatus;
+
+                if (status == String.Empty)
+                {
+                    return "hidden";
+                }
+
+                var cssClass = status.ToLower();
+                cssClass = cssClass.Replace(" ", "");
```

Comment:
do you need to care about tabs? Surely it'll be better to use: cssClass.Trim()

Reply:
This code was used to map statuses like In Progress and Not Required to their relevant CSS class names. I've rewritten this code so that it's easier to see what the mappings are